db/sql/V3__add_last_update_time_for_activities.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE activities ADD last_update_time DATETIME;
```

Comment:
This is good! In the future, we should think about adding the code which uses this when we add it as well :).